### PR TITLE
Add The Option To Reverse Admin Search Queries

### PR DESCRIPTION
**Pull request by sampoder**
I believe this fixes #3550; @maxwofford & @MelSmith104 is this what you were looking for? I think this let's you do a query that would return all accounts that aren't hardware accounts:

### CODE REVIEW

File: app/controllers/admin_controller.rb [hunk 1]
```diff
@@ -1175,6 +1176,13 @@ def filtered_events(events: Event.all)
     states << "approved" if @approved
     states << "rejected" if @rejected
     relation = relation.where("aasm_state in (?)", states)
+
+    if @inverse
+      relation = Event.all.where.not(id: relation.pluck(:id))
+    else
+      relation
+    end
```
Comment (garyhtou) [inline app/controllers/admin_controller.rb]: check this out!
https://blog.saeloun.com/2021/04/21/rails-7-invert-where-method/

Comment (sampoder) [reply app/controllers/admin_controller.rb]: That's sick! So would that look like turning:

```
@events = filtered_events.page(@page).per(@per).reorder(Arel.sql("COALESCE(events.activated_at, events.created_at) desc"))
```

into when inverted is selected:

```
@events = filtered_events.invert_where.page(@page).per(@per).reorder(Arel.sql("COALESCE(events.activated_at, events.created_at) desc"))
```

**Comment by maxwofford** [comment]
To clarify the original issue, we're looking for something like this:

Screenshot 2023-08-18 at 13 16 46

Just like how funded can be "Funded", "Not funded", or "Doesn't matter if funded or not"

**Comment by sampoder** [comment]
Alrighty! I made a checkbox thing for events:

Screenshot 2023-08-18 at 7 03 04 PM

**Comment by sampoder** [comment]
my only worry with the checkboxes is that selecting them all is a pain. maybe I could add an option that unselects them all. thoughts?

**Comment by sampoder** [comment]
Is this still of interest to people or should I close this?

**Pull request closed without merging**